code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -7,7 +7,7 @@
 	ammo_x_offset = 3
 
 /obj/item/weapon/gun/energy/shock_revolver
-	name = "shock revolver"
+	name = "tesla revolver"
 	desc = "A high-tech revolver that fires internal, reusable shock cartridges in a revolving cylinder. The cartridges can be recharged using conventional rechargers."

```

Comment:
You should update the description to reflect that this weapon is an entirely different gun now, and what it does. If you do not mind, I think I am going to ask @WJohn to provide a new sprite (and probably call it a rifle) for it as well. It is a lightning gun at this point.